lib/ansible/cli/config.py

```diff
@@ -1,4 +1,4 @@
-# Copyright: (c) 2017, Ansible Project
+# Copyright: (c) 2017-2018, Ansible Project
```

Comment:
I'd leave the hyphenating off.  I believe we have a doc somewhere that indicates to just use the start year. Otherwise this becomes hard to manage, and we may forget to advance this.

Reply:
Thanks!  I must have skipped that paragraph the last time I read the licensing advice doc.

Reply:
Fixed to only have 2017.